cytosnake/helpers/helper_funcs.py

```diff
@@ -204,6 +202,23 @@ def consensus_output() -> str:
     return str(results_path / f"{output_name}.{ext}")
 
 
+def parquet_output():
```

Comment:
it seems to me that the only purpose of this file is to append a hardcoded `.parquet` to `output_name`.

This to me smells too complex and you might benefit (in regards to maintainability) from thinking through a simpler solution

Reply:
Yeah I've noticed that too as well. This is mentioned in [#32](https://github.com/WayScience/CytoSnake/issues/32)

I will be creating a seperate PR that will fix this issue.